README.md

```diff
@@ -1 +1,287 @@
-# ballerina-gke-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-gke-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-gke-deployment)
+
+# Ballerina Deployment with Google Cloud Platoform Kubernetes Engine
+
+[Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/) is a powerful cluster manager and orchestration system for running your Docker containers
+
+> In this guide you will learn about building a Ballerina service and deploying it on Google Kubernetes Enginer (GKE).
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Deployment](#deployment)
+- [Testing](#testing)
+
+## What you’ll build
+
+In this guide, you will build a simple Ballerina Hello World service, and you will deploy this service on GKE.
+
+## Compatibility
+
+| Ballerina Language Version
+| --------------------------
+| 0.990.0
+
+## Prerequisites
+
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE
+- [Docker](https://docs.docker.com/engine/installation/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Docker Hub Account](https://hub.docker.com/)
+- [Google Cloud Platform account](https://cloud.google.com/)
+
+### Optional requirements
+
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina))
+
+## Implementation
+
+### Create the project structure
+
+Ballerina is a complete programming language that can have any custom project structure that you wish. Although the language allows you to have any module structure, use the following module structure for this project to follow this guide.
+```
```

Comment:
Add a line break between text and the back ticks. There are some rendering issues that occur sometimes when this line break does not exist.